src/models/AppState.js

```diff
@@ -143,6 +143,7 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
     local: {
       baseUrl: 'string',
       lastAuthResponse: ['object', true, {}],
+      profileSchema: ['object', true, {}],
```

Comment:
I don't think we need this attribute. From what I can see, we are storing a full api response here, which we already have in `lastAuthResponse`.